scylla_pg_core/src/config.rs

```diff
@@ -45,7 +45,12 @@ impl PGConfig {
     }
     pub fn to_without_db_config(&self) -> tokio_postgres::Config {
```

Comment:
Why do we need this method with `hardcoded db`? The only difference between this method and `to_pg_config` is the database is hardcoded. Why do we hardcode `postgres`as the db when we are migrating?

Reply:
When there is no db mentioned in the connection string, the api will try to connect with DB name as the username instead if there is a maintenance db name mentioned to api to connect to the host and then create DB then there is no error. Using the default maintenance db name for creating connection to the host before creating DB is recommended way.